recipes/ms-gsl/all/conanfile.py

```diff
@@ -1,8 +1,12 @@
 import os
-from conans import ConanFile, tools
-from conans.errors import ConanInvalidConfiguration
 
-required_conan_version = ">=1.43.0"
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import get
+from conan.tools.scm import Version
+
+required_conan_version = ">=1.47.0"
```

Comment:
```suggestion
required_conan_version = ">=1.50.0"
```
 from conan.tools.build import check_min_cppstd  requires 1.50.0

Reply:
Done!